locations/spiders/du_pareil_au_meme.py

```
@@ -0,0 +1,24 @@
+
+class DuPareilAuMemeSpider(SitemapSpider, StructuredDataSpider):
+    name = "du_pareil_au_meme"
+    item_attributes = {
+        "brand": "Du Pareil au Même",
+        "brand_wikidata": "Q3040318",
+    }
+    sitemap_urls = ["https://boutiques.dpam.com/sitemap_pois.xml"]
+    wanted_types = ["ClothingStore"]
+    sitemap_rules = [
+        (r"/en/", "parse_sd"),
+    ]
```

Ruff (0.16.2)

[warning] Mutable default value for class attribute

(RUF012)